### PR TITLE
added a few different examples 

### PR DESCRIPTION
**Pull request by padraicbc**

### CODE REVIEW

File: setoperators.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 c = set(['a', 'c'])
 
 # Intersection
-print(a & b)
```
Comment (brennerm) [inline setoperators.py]: Why removed?

Comment (brennerm) [inline setoperators.py:8]: why removed?

**Comment by brennerm** [comment]
Have a look at my note please.

File: setoperators.py (hunk 1 above)
Comment (padraicbc) [inline setoperators.py:8]: Sorry, accidental deletion.

**Comment by brennerm** [comment]
No problem. Want to add yourself to the contributors?

**Comment by padraicbc** [comment]
Sure, I will add a few more bits today also.

File: copylist.py [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+"""a fast way to make a shallow copy of a list"""
 
 a = [1, 2, 3, 4, 5]
 print(a[:])
+
+"""using the list.copy() method (python3 only)"""
+
+a = [1, 2, 3, 4, 5]
+
+print(a.copy())
```
Comment (brennerm) [inline copylist.py:11]: is it really a trick using a copy function?

Comment (padraicbc) [inline copylist.py:11]: Explicit is better than implicit, the copy method was added in python 3.

**Pull request merged**